tests/test_persona_system.py

```
@@ -0,0 +1,309 @@
+"""
+AIペルソナシステムのテストスイート
+"""
+import pytest
+from datetime import datetime
```

Ruff (0.12.2)

`datetime.datetime` imported but unused

Remove unused import: `datetime.datetime`

(F401)